.gitignore

```diff
@@ -1,5 +1,7 @@
+node_modules
 /lib/
 /es/
-/node_modules/
-/built/
+/esm/
```

Comment:
I went ahead and changed the `esm` directory to `esm` to help clarify it's purpose.   @jayphelps thanks for describing this for me.